src/Tools/NodeDocumentationMarkdownGenerator/NodeDocumentationMarkdownGenerator.csproj

```diff
@@ -23,6 +23,7 @@
         <PackageReference Include="System.Memory" Version="4.5.4" />
         <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
         <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.3" />
+        <PackageReference Include="System.Drawing.Common" Version="6.0.0" Condition=" '$(TargetFramework)' == 'net6.0' " />
```

Comment:
whats this for?

Reply:
Well, NodeDocumentationMarkdownGenerator has System.Drawing references so it needs the System.Drawing dll
I am not sure if this tool can run outside of Dynamo or not. If it can run outside of dynamo (separate process) then it needs to have its own System.Drawing.Common reference